java/developer-guide/working-with-outlook-items/working-with-recurrences/_index.md

```diff
@@ -0,0 +1,527 @@
+---
+title: Working with Recurrences
+type: docs
+weight: 140
+url: /java/working-with-recurrences/
+---
+
+
+## **Working with Daily Recurrences**
+Aspose.Email supports creation of daily recurrences using MapiCalendarDailyRecurrencePattern. Three different Mapi calendar recurrence end types can be used including EndAfterNOccurrences, EndAfterDate and NeverEnd. This section demonstrates the creation of different daily recurrence patterns.
```

Comment:
`Aspose.Email supports the creation of daily recurrences using **MapiCalendarDailyRecurrencePattern**. Three different Mapi calendar recurrence end types can be used including **EndAfterNOccurrences**, **EndAfterDate** and **NeverEnd**. This section demonstrates the creation of different daily recurrence patterns.`